tools/logger/convert.c

```diff
@@ -517,7 +520,7 @@ static void print_entry_params(const struct log_entry_header *dma_log,
 			raw_output && strlen(ids) ? "-" : "",
 			ids);
 		if (time_precision >= 0)
-			fprintf(out_fd, time_fmt,
+			fprintf(out_fd, "%s", time_fmt,
 				to_usecs(dma_log->timestamp - timestamp_origin), dt);
```

Comment:
time_fmt seems to itself be a format specifier; you're changing semantics here unfortunately.